src/main/java/org/testng/reporters/IReporterConfig.java

```diff
@@ -0,0 +1,4 @@
+package org.testng.reporters;
+
+public interface IReporterConfig {
```

Comment:
Same case here as well. Please help add some java docs to this marker interface.

Reply:
This marker doesn't add any value and is being removed.

Reply:
Just an idea: what if we add a setProperty method here?

Reply:
The marker interface has been removed.